### ISSUE TITLE
Main menu accessibility improvements

### ISSUE THREAD

**Issue by andrewmarino** [unitymakesus/tcf#8]
- [x] Mobile menu trigger should ideally be `<button>` element or an `<a role="button">`. The button should also have the following aria attributes: aria-expanded, aria-controls to reference its status and dropdown element it controls to screen readers. Currently it is a checkbox form input.
- [x] Tabindex should not be programatically set when menu is opened and closed on mobile. This can be avoided by using `display: none`.
- [x] Menus and submenus should be opened by their toggles, not focus.
